wled00/NodeStruct.h

```diff
@@ -23,7 +23,13 @@ struct NodeStruct
   String    nodeName;
   IPAddress ip;
   uint8_t   age;
-  uint8_t   nodeType;
+  union {
```

Comment:
IMO it is not a huge deal "wasting" space here, I wouldn't even add alternative type IDs starting from 1 at all, just adds overhead, as we should keep the 82/32-35 IDs anyways for compatibility.
We still have an extra byte for future enhancements without increasing the size of `NodeStruct` as 24 bit is sufficient for all build numbers until the year 2167 😄

And don't underestimate how many ESP32 variants Espressif can still throw on the market (H6? C6? C2? Hard to keep track...). Maybe we even want to distinguish different flash layouts or something with this in the future, so 7 bits seems reasonable for type.